Src/Card/Card.cpp

```diff
@@ -1 +1,136 @@
-#include "Card.h"
\ No newline at end of file
+#include "Card.h"
+//Constructor
+Card::Card(){
+
+}
+Card::Card(const string &IdAccount, const string &IdHolder, const string &pin, const int &balance)
```

Comment:
Nên validate trước khi gán.
```suggestion
Card::Card(const string &IdAccount, const string &IdHolder, const string &pin, const int &balance)
```